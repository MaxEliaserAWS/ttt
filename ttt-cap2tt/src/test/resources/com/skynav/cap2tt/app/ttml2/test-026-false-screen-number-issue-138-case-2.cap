Lambda字幕V4	DF0+1	SCENE"和文標準"

283	00135601/00135912	Ａ 75本
				Ｂ 1000本
