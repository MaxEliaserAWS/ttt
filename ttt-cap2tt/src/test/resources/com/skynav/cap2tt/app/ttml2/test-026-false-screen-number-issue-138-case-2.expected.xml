<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:profile="http://www.w3.org/ns/ttml/profile/ttml2-presentation" ttva:model="ttml2" xml:lang="ja" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" tts:textAlign="center"/><style xml:id="s2" tts:textAlign="start"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:13:56:01" end="00:13:59:12" style="s1"><span style="s2">Ａ 75本<br/>Ｂ 1000本</span></p></div></body></tt>